ldaptor/test/test_encoder.py

```diff
@@ -6,24 +6,50 @@
 
 import six
 
-from ldaptor.encoder import to_bytes
+from ldaptor.encoder import to_bytes, WireStrAlias
 
 
 class WireableObject(object):
+    """
+    Dummy object with bytes representation
```

Comment:
Instead of using "dummy" maybe is better to describe what it actually does.

Like, `always returns the same value for toWrite` or something like that :)

Or `Implementation with constant toWire value`

Reply:
Made more verbose docstring for it